packages/fxa-auth-server/test/local/payments/stripe.js

```diff
@@ -4626,22 +4542,21 @@ describe('StripeHelper', () => {
         'setCustomerLocation',
         {
           customer: { id: customer1.id },
+          postalCode: expectedAddressArg.postalCode,
+          country: expectedAddressArg.country,
         }
       );
       sinon.assert.calledOnceWithExactly(Sentry.captureException, err);
     });
 
     it('fails when an error is thrown while updating the customer address', async () => {
-      sandbox
-        .stub(stripeHelper, 'extractCustomerDefaultPaymentDetails')
-        .resolves({ postalCode: '99999', country: 'GD' });
       sandbox.stub(stripeHelper, 'updateCustomerBillingAddress').rejects(err);
-      const actual = await stripeHelper.setCustomerLocation(customer1);
+      const actual = await stripeHelper.setCustomerLocation({
+        customerId: customer1.id,
+        postalCode: expectedAddressArg.postalCode,
+        country: expectedAddressArg.country,
+      });
       assert.isFalse(actual);
-      sinon.assert.calledOnceWithExactly(
-        stripeHelper.extractCustomerDefaultPaymentDetails,
-        customer1
-      );
```

Comment:
Can we assert that Sentry.withScope and scope.setContext are called with the expected values here?